modules/manually-gathering-logs-with-SSH.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// *installing/installing-gather-logs.adoc
+
+[id="installation-manually-gathering-logs-with-SSH_{context}"]
+= Manually gathering logs with SSH access to your host(s)
+
```

Comment:
If there was something we could say before the Prerequisites about why or what situation you might want to delve in deeper manually vs just the tool-gathered logs, would be nice, but maybe not critical.

Reply:
@adellape Good point. @sferich888 What is the main purpose of gathering manually?

Reply:
You want to manually run these commands in situations where must-gather or automated Collection methods don't work.